site-src/api-types/httproute.md

```diff
@@ -43,6 +43,64 @@ spec:
 Note that the target Gateway needs to allow HTTPRoutes from the route's
 namespace to be attached for the attachment to be successful.
 
+You can also attach routes to specific sections of the parent resource.
+For example, let's say that the `acme-lb` Gateway includes the following
+listeners:
```

Comment:
This looks like it was also accidentally included from another PR, some rebasing/git history cleanup may be needed here.